packages/util-dev/src/util.ts

```diff
@@ -1,2 +1,4 @@
 export const flushPromises = (setImmediate = global.setImmediate) =>
   new Promise((resolve) => setImmediate(resolve, void 0));
+
+export const sleep = (ms: number) => new Promise((resolve) => setTimeout(resolve, ms));
```

Comment:
We're already using [delay](https://www.npmjs.com/package/delay) package for that

Reply:
f3e0b8296 and de03eca3c should fix this